packages/web-components/fast-components/src/utilities/apply-mixins.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Apply mixins to a constructor.
+ *
+ * Shamelessly stolen from https://www.typescriptlang.org/docs/handbook/mixins.html
```

Comment:
We should add an acknowledgements.md file to fast-components and call this out. I've got one in fast-element that calls out influences, etc.